source/mod_ffield.f90

```diff
@@ -394,9 +386,8 @@ subroutine ffield_mod_link(iErr)
       ffNLn = ffNLn-1
       ffMSn = ffMSn-1
       ffNLFile = ffNLFile-1
-      gamma0r=pmap/e0 !      gamma0=e0/pmap
-      beta0=sqrt(one-gamma0r*gamma0r) ! = sqrt(one-one/(gamma0*gamma0))
-!      p0=beta0*e0*c1e6/clight
+      gamma0r=pmap/e0 
+      beta0=sqrt(one-gamma0r*gamma0r)
```

Comment:
Hmm, just noticed this.

First of all, do not use `pmap` to compute gamma. `pmap` is NOT the mass of the reference particle. The variable `nucm0` is. `pmap` is a constant that may or may not be used to set `nucm0`, depending on the input file settings.

Secondly, both the variables for gamma0, beta0 and gammar already exist in common modules.

Reply:
Ok I will check them and make modification

Reply:
They are all in module `mod_common`, so should be straight forward to use.

Reply:
Done